### DISCUSSION TITLE
Chest Mimic as Copy Source

### DISCUSSION THREAD

**Discussion by plinknato** [loathers/garbage-collector#1790] [Ideas]
It would seem to me that Chest Mimic eggs are a viable copy source for Knob Goblin Embezzelers, but with a couple of draw backs:

**1. Familiar EXP Charge:** Gathering the 550 familiar exp (fxp) for the 11 copies would take a number of turns. From my rough calculations this would require 20 turns with Feeling Fancy and Blue Swayed plus very affordable equipment (3 Teacher's Pens and Giant Yellow Hat), and can be cut further from there.

**2. Familiar Slot:** Check Mimic would need to be active to generate an egg, consuming the bonus from familiar and equipment plus associated bonuses.

There is a trade off with the second and first, in that 100 fxp can be used at the Mimic DNA Bank to extract an egg. I am not in a position to relatively value this trade off and will leave this to those with more knowledge.

At the end of the day with a 4:1 advantge in base meet between Embezzler and Barf fights it would seem profitable to leverage these copies as they would take less then 3 turns to generate. Does this seem like a viable path or is it to marginal to be worth implementing? If there is more documentation on the math to calculate expected yield please point me in that direction?

**Comment by gausie** [comment]
This was not attempted in the past because the API for KoL did not return familiar experience over 400. As of a few days ago it does, so someone will get to this eventually. Sounds like you've given this some thought so if you'd be up for writing a pull request yourself, I'm sure many people would be on hand to help you out and give feedback.

**Comment by pstalcup** [comment]
As a reference, here is what I scoped out as "ideal" mimic behavior (I am not sure its actually ideal but that is what I called it)

ok so, just to be clear an ideal mimic implementation looks like this:
* whether or not you are  ascending, use as many eggs as you can at the start of the day.
* try to produce 11 eggs while fighting embezzlers.
* if you can produce more eggs:
  * if you have digitize, produce all your eggs against the first digitized kge where you have (egg productions left) * 50 exp
    * If you are ascending, fight them immediately
    * if you are not ascending, save them for tomorrow
  * if you don't have digitize, as soon as you have (egg productions left) * 50 + 50 exp, eggfax an embezzler and generate all remaining eggs
    * if you are ascending, fight them immediate
    * if you are not ascending, save them for tomorrow
    
Implicit in this is assuming that we value the chest mimic as having 1/50th of an embezzler fight for each famxp you generate

**Comment by Rinn** [comment]
this was added in may e1c7d0ab01396ade8260dc92eefb437178dee3e6
